tests/test_definitions/other_expectations/expect_this_test_to_be_suppressed.json

```diff
@@ -1,21 +1,22 @@
 {
   "expectation_type": "expect_column_to_exist",
-  "datasets":[
+  "datasets": [
     {
-      "data": {"b": [1, 2, 3, 4]},
-      "tests":
-        [{
+      "data": { "b": [1, 2, 3, 4] },
+      "tests": [
+        {
           "title": "This test should never run!",
           "exact_match_out": false,
           "suppress_test_for": ["pandas", "sqlalchemy", "spark"],
-          "in": ["a"],
-          "kwargs": {},
+          "in": { "column": "a" },
```

Comment:
Is this change to accomodate the rendering issue flagged above? Because the test is designed to be input to an expectation (vs a renderer)...